test-e2e/manager-basic.js

```diff
@@ -13,17 +14,49 @@ test('Managing multiple projects', async (t) => {
     'no projects exist when manager is initially created'
   )
 
-  const createdProjectIds = [
-    await manager.createProject(),
-    await manager.createProject(),
-    await manager.createProject(),
-  ]
+  const createdProjectId = await manager.createProject({
+    name: 'created project',
+  })
 
-  const allProjects = await manager.listProjects()
+  const addedProjectId = await manager.addProject({
+    projectKey: KeyManager.generateProjectKeypair().publicKey,
+    encryptionKeys: { auth: randomBytes(32) },
+    projectInfo: { name: 'added project' },
+  })
 
-  t.is(allProjects.length, createdProjectIds.length)
-  t.ok(
-    allProjects.every((p) => createdProjectIds.includes(p.projectId)),
-    'all created projects are listed'
+  const listedProjects = await manager.listProjects()
+
+  t.is(listedProjects.length, 2)
+
+  t.is(
+    listedProjects[0].projectId,
```

Comment:
Why would this always be the first entry in this array? Should it be? Or should we just test that one of the entries has this projectId? If we are sorting by date then we can't guarantee this will always be first, if they are both written in the same millisecond.

Reply:
there's no sorting that's happening at the moment. think we can decide on that in a separate PR. will update the test to just look for the id rather than assuming ordering 👍

Reply:
addressed via https://github.com/digidem/mapeo-core-next/pull/215/commits/3fd174dfd1be6318719c444a062146fa5d862e12